soc/riscv/riscv-privilege/andes_v5/Kconfig.defconfig.ae350

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2021 Andes Technology Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_RISCV_ANDES_AE350
+
+config SOC
+	string
+	default "ae350"
+
+config SYS_CLOCK_TICKS_PER_SEC
+	default 100 if !CACHE_ENABLE
```

Comment:
Why does this depend on `!CACHE_ENABLE`? From what I've checked, the value will default to 100 anyway if `TICKLESS_KERNEL` is set to `n`, and to 10000 otherwise.

Reply:
Because we recommand 100 tick rate in cache disabled mode no matter it's ticked or tickless kernel.

Disabling cache affects system performance much. It can break few kernel testcase (e.g. `tests/kernel/sched/schedule_api`) in the high tick rate because the checking of these cases is based on the tick unit. We'd still like to keep all kernel testcases passed when disabling cache.

The reason why we provide cache disabled mode is that it's helpful to reduce the system complexity of cache (especially DMA coherence) in the development stage or debugging.

Reply:
If there isn't any problem, I will resolve this comment.

Reply:
Thanks for the explanation, makes sense. You can resolve this comment. :)